h2o-extensions/target-encoder/src/main/java/ai/h2o/targetencoding/TargetEncoderBuilder.java

```diff
@@ -64,8 +61,16 @@ public void computeImpl() {
     }
 
 
-    private String[] encodedColumnsFromIgnored(final String[] minued, final String[] subtrahend, final String responseName,
-                                                      final String foldColumnName) {
+    private String[] encodedColumnsFromIgnored(final Frame trainingFrame, final String[] subtrahend, final String responseName,
+                                               final String foldColumnName) {
+      // Ignore non-categorical columns by default
+      final ArrayList<String> minued = new ArrayList();
+      for (int i = 0; i < trainingFrame.numCols(); i++) {
+        if (trainingFrame.vec(i).isCategorical() && !trainingFrame.name(i).equals(responseName)) {
```

Comment:
duplicate logic with response: it's filtered out in `minued` and then removed in `subtrahend`.
Would be easier to keep `minued` = trainingFrame._names, and then specify all the things you remove.